nifi-nar-bundles/nifi-aws-bundle/nifi-aws-processors/src/main/java/org/apache/nifi/processors/aws/kinesis/stream/ConsumeKinesisStream.java

```diff
@@ -481,25 +525,25 @@ public void onScheduled(ProcessContext context) {
 
     @Override
     public void onTrigger(final ProcessContext context, final ProcessSessionFactory sessionFactory) {
-        if (worker == null) {
+        if (scheduler == null) {
             synchronized (WORKER_LOCK) {
```

Comment:
Should we change all `worker` references to `scheduler` for consistency?

Reply:
You mean like `workerId` and `workerState`?  I left these because the library still refers to them by the same name.

Reply:
Think I noted the same after further reviewing the code/changes/docs, so happy to not confuse things further than already may be the case with the AWS SDK itself